iam-login-service/src/test/java/it/infn/mw/iam/test/service/client/ClientManagementServiceTests.java

```diff
@@ -141,7 +142,7 @@ public void testClientRetrieve() {
     RegisteredClientDTO client = managementService.retrieveClientByClientId("client").orElseThrow();
 
     assertThat(client.getClientId(), is("client"));
-    assertThat(client.getClientSecret(), is("secret"));
+    assertThat(new BCryptPasswordEncoder(12).matches("secret", client.getClientSecret()), is(true));
```

Comment:
Usare la funzione `bcrypt()`. Ma non manca un `encode(secretToEncode)`?
Cosa fa quel `matches` fa in automatico l'encoding della stringa `"secret"`?